WordPress-VIP-Go/ruleset.xml

```diff
@@ -89,20 +89,24 @@
 	 -->
 	<rule ref="WordPress.Security.EscapeOutput.OutputNotEscaped"/>
 	<!-- Should fix all of them but it doesn't need a manual review -->
-	<rule ref="WordPress.WP.AlternativeFunctions.file_system_read_fclose">
-		<type>error</type>
-		<message>File operations should use WP_Filesystem methods instead of direct PHP filesystem calls. Found: %s(). Read more here: https://vip.wordpress.com/documentation/using-wp_filesystem-instead-of-direct-file-access-functions/</message>
+	<rule ref="WordPress.WP.AlternativeFunctions.file_system_read">
+		<type>warning</type>
+		<message>File operations only work using the VIP stream on VIP Go. Make sure you are getting the proper base directory using wp_upload_dir() when using functions such as %s() or it will return unexpected results. Read more here: https://vip.wordpress.com/documentation/using-wp_filesystem-instead-of-direct-file-access-functions/</message>
 	</rule>
 	<rule ref="WordPress.WP.AlternativeFunctions.file_system_read_fopen">
-		<type>error</type>
-		<message>File operations should use WP_Filesystem methods instead of direct PHP filesystem calls. Found: %s(). Read more here: https://vip.wordpress.com/documentation/using-wp_filesystem-instead-of-direct-file-access-functions/</message>
+		<type>warning</type>
+		<message>File operations only work using the VIP stream on VIP Go. Make sure you are getting the proper base directory using wp_upload_dir() when using functions such as %s() or it will return unexpected results. Read more here: https://vip.wordpress.com/documentation/using-wp_filesystem-instead-of-direct-file-access-functions/</message>
+	</rule>
+	<rule ref="WordPress.WP.AlternativeFunctions.file_system_read_fclose">
+		<type>warning</type>
+		<message>File operations only work using the VIP stream on VIP Go. Make sure you are getting the proper base directory using wp_upload_dir() when using functions such as %s() or it will return unexpected results. Read more here: https://vip.wordpress.com/documentation/using-wp_filesystem-instead-of-direct-file-access-functions/</message>
 	</rule>
 	<rule ref="WordPressVIPMinimum.JS.HTMLExecutingFunctions.append">
 		<type>error</type>
 	</rule>
 	<rule ref="WordPressVIPMinimum.Performance.FetchingRemoteData.FileGetContentsUnknown">
-		<type>error</type>
-		<message>%s() is uncached. If this is being used to query a remote file please use wpcom_vip_file_get_contents() instead. If it's used for a local file please use WP_Filesystem instead. Read more here: https://vip.wordpress.com/documentation/using-wp_filesystem-instead-of-direct-file-access-functions/</message>
+		<type>warning</type>
+		<message>%s() is uncached. File operations only work using the VIP stream on VIP Go. Make sure you are getting the proper base directory using wp_upload_dir() or it will return unexpected results. Read more here: https://vip.wordpress.com/documentation/using-wp_filesystem-instead-of-direct-file-access-functions/</message>
```

Comment:
I think this one probably needs a bit more refinement since it changes the original intent of the error. There are three cases:

- local file
- file in the media library (via stream)
- remote file

Interestingly, `wpcom_vip_file_get_contents` does not work with a `vip://` stream URL, so we should revert back to the original message but with a slight tweak:

```suggestion
		<message>%s() is uncached. If the function is being used to fetch a remote file (e.g. a URL starting with https://), please use wpcom_vip_file_get_contents() to ensure the results are cached. For more details, please see https://wpvip.com/documentation/vip-go/fetching-remote-data/</message>
```